src/components/DownloadComponents/DetailsCompleted/DetailsCompleted.tsx

```diff
@@ -1,86 +1,79 @@
-import { FC, useEffect, useState } from "react";
+import { FC } from "react";
 
-import { Heading, P } from "../../Typography";
-import Button from "../../Button";
-import Box from "../../Box";
-import Flex from "../../Flex";
-import Icon from "../../Icon";
+import Typography, { P } from "@/components/Typography";
+import Button from "@/components/Button";
+import Flex from "@/components/Flex";
+import Box from "@/components/Box";
 
 export type DetailsCompletedProps = {
   email?: string;
   school?: string;
   onEditClick: () => void;
 };
 
+const getSchoolName = (school: string | undefined) => {
+  if (school === "notListed") {
+    return "My school isn’t listed";
+  } else if (school === "homeschool") {
+    return "Homeschool";
+  } else {
+    return school;
+  }
+};
+
 const DetailsCompleted: FC<DetailsCompletedProps> = ({
   email,
   school,
   onEditClick,
 }) => {
-  const getSchoolName = (school: string | undefined) => {
-    if (school === "notListed") {
-      return "My school isn’t listed";
-    } else if (school === "homeschool") {
-      return "Homeschool";
-    } else {
-      return school;
-    }
-  };
-
-  const [displayEmail, setDisplayEmail] = useState(false);
-  const [displaySchool, setDisplaySchool] = useState(false);
-
-  useEffect(() => {
-    if (email) {
-      setDisplayEmail(true);
-    }
-
-    if (school) {
-      setDisplaySchool(true);
-    }
-  }, [email, school]);
-
   return (
-    <Box $mt={56}>
-      <Flex $mb={12}>
-        <Heading tag="h3" $font={"heading-5"}>
-          Details complete
-        </Heading>
-        <Icon
-          variant="brush"
-          name={"tick"}
-          size={36}
-          $background={"success"}
-          $ml={28}
+    <Box $width={["100%", 420]} $height={"max-content"} $position="relative">
+      <Box
+        $position="absolute"
+        $top={0}
+        $left={0}
+        $width="100%"
+        $height="100%"
+        $zIndex="behind"
+      >
+        <svg
```

Comment:
Would the <BrushBorders /> component would be a better choice if possible?

Reply:
aha yeah, BrushBorders works perfectly, thanks!!